core/classes/Core/Util.php

```diff
@@ -283,6 +283,22 @@ public static function replaceAnchorsWithText(string $data): string {
         return URL::replaceAnchorsWithText($data);
     }
 
+    private static function getSettingsCache(?string $module): ?array {
+        $cache_name = $module !== null ? $module : 'core';
```

Comment:
Here we default to module name of `'core'`, but in `Util::getSetting` we default to null as the module name for setting lookup.
I think we should use 1 convention

Reply:
Does php support null keys for dictionaries? I just assumed it didn't

Reply:
Apparently it does support null keys but it has weird semantics (actually stored as an empty string). Probably best to avoid.